sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -167,15 +167,19 @@ object Klaviyo {
      * @param pushToken The push token provided by the device push service
      */
     fun setPushToken(pushToken: String) = safeApply {
-        Registry.dataStore.store(EventKey.PUSH_TOKEN.name, pushToken)
```

Comment:
What was our reason for popping push tokens into a different store originally? `UserInfo` was just initially designed to store top-level identifiers?

Reply:
Yeah, and we were trying to separate push functionality from other analytics

Reply:
At one point I think the whole setpushtoken function was defined in the push model even

Reply:
Ah gotcha, thanks for the context. Makes sense